templates/2_Cluster_Pixels.ipynb

```diff
@@ -116,6 +116,37 @@
     "# fovs = [\"fov14\"]"
```

Comment:
<p>I would add a comment about the type of error they might get when using multiprocessing so that as a first troubleshooting step, they can try switching this back. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>